include/level_commands.h

```diff
@@ -253,8 +253,8 @@ enum GoddardScene {
     CMD_PTR(NULL), \
     CMD_PTR(NULL)
 
-#define LOAD_YAY0(seg, romStart, romEnd) \
-    CMD_BBH(LEVEL_CMD_LOAD_YAY0, 0x0C, 0x0000), \
+#define LOAD_YAZ0(seg, romStart, romEnd) \
+    CMD_BBH(LEVEL_CMD_LOAD_YAZ0, 0x0C, 0x0000), \
```

Comment:
What if instead you made the LOAD_YAY0 macro point to LOAD_YAZ0? This would solve fast64's issues, but would still be merge conflict hell for a grand majority of hacks if you keep the level script changes.